Index: gnome-shell-40.5/js/ui/main.js
===================================================================
--- gnome-shell-40.5.orig/js/ui/main.js
+++ gnome-shell-40.5/js/ui/main.js
@@ -393,6 +393,12 @@ function _getDefaultStylesheet() {
     // theme
     if (St.Settings.get().gtk_theme == 'HighContrast')
         stylesheet = _getStylesheet(name.replace('.css', '-high-contrast.css'));
+    if (St.Settings.get().gtk_theme == 'Yaru-dark')
+        stylesheet = _getStylesheet(name.replace('.css', '-dark.css'));
 
     if (stylesheet == null)
         stylesheet = _getStylesheet(sessionMode.stylesheetName);
